src/pages/main/components/selected-dataset/dataset-delete-dialog.tsx

```diff
@@ -0,0 +1,33 @@
+import { ReactElement } from 'react'
```

Comment:
Why make a component for this? It will be used only in one place

Reply:
Fixed